code/modules/cargo/exports/large_objects.dm

```diff
@@ -110,6 +110,7 @@
 	var/obj/machinery/portable_atmospherics/canister/C = O
 	var/worth = 10
 	var/gases = C.air_contents.gases
+	var/gas_elasticity = 0.00033
```

Comment:
why not just use k_elasticity?

Reply:
k_elasticity should be used for each canister, doesn't it?

Reply:
?

Reply:
ah nothing this overrides the get_cost() proc, so each canister just get sold at full price, regardless. i could modify the k_elasticity, but in case of a refractor is better to distinguish the canister elasticity and the gas one

Reply:
what? the canister itself has no inherent value

Reply:
if this is refactored and they want to make the can have separate value and elasticity on its own they should add that in the refactor. Someone wont know this is intentional in the future.